cumulusci/core/utils.py

```diff
@@ -96,13 +98,20 @@ def process_glob_list_arg(arg):
 
 def process_list_arg(arg):
     """Parse a string into a list separated by commas with whitespace stripped"""
+    if isinstance(arg, Path):
+        arg = str(arg)
```

Comment:
@prescod can you help me understand this addition? I'm failing to think of a scenario where a Path instance would end up in a task option parsed from the CLI or loaded from yaml.

Reply:
Tasks can be loaded from CLI or YAML but this code is also executed when they are called as an API.

Unit tests are an example, but other places we call them as an API are the `snowfakery` task which calls sub-tasks and Robot keywords that create tasks.

Reply:
@prescod ok. I like the idea of making options that accept string paths also accept pathlib instances, but this seems incomplete or maybe the wrong place to do it since it only achieves that for options that use parse_list_arg. I'm okay with leaving this in but it seems like something that should be addressed as part of switching to pydantic validation of options.

Reply:
So is it okay if I leave it until we do the Grand Options Refactoring, because it will add risk to this PR if I take it out.